packages/styles/components/_c.stars-input.scss

```diff
@@ -0,0 +1,59 @@
+.mc-stars-input {
+  $parent: &;
+
+  display: flex;
+
+  &__label {
+    color: $color-star-full;
+    cursor: pointer;
+    position: relative;
+  }
+
+  &__icon {
+    @include set-stars-size($stars-medium);
+
+    fill: currentColor;
+  }
+
+  & &__radio:checked ~ &__label {

```

Comment:
no need for that, `:checked ~ &__label` should be enought, same for the following

Reply:
I kept the value `&__radio:checked ~ &__label`, because it's associated with `&__radio:focus ~ &__label`. Therefore, to keep consistency between these two selectors that apply to the same element, I kept the writing `&__radio` for both cases.